### ISSUE TITLE
repAssess: representativeness value is based on estimated asymptote

### ISSUE THREAD

**Issue by allisonglider** [BirdLifeInternational/track2kba#40]
First, thank you so much for developing this useful package! I have a question about the error: "Estimated asymptote differs from target; be aware that representativeness value is based on estimated asymptote (i.e. est_asym)." I understand why I get this error message, but I am a bit confused about why the function defaults to using the estimated asymptote rather than the target asymptote if the estimated asymptote is lower than the target. To me, this provides an inflated estimate of representativeness and makes it harder for a user to interpret the output. 

For example, I have tracking data from a number of seabird breeding colonies that I would like to define important at-sea regions for at the 95% UD level. For some sites with smaller sample sizes, I get this error where my target asymptote is 95% but the estimated asymptote is 65%. My max pred value from the bootTable is 0.616, so if I calculate representativeness (using the equation from the Beal et al 2021 supplement) for the target asymptote  it is 0.65 but for the estimated asymptote it is 0.866. This makes it appear that my tracks are representative, but they clearly are not representative.

I can see why it would be necessary to use the estimated asymptote if this value is higher than the target, but I don't understand why representativeness is calculated this way if the estimated asymptote is lower than the target. Can you explain this?

**Comment by MartinBeal** [comment]
Hey Allison,

The warning message you refer to exists for precisely this situation. I.e., when the estimated asymptote does not appear to be the 'real' asymptote. You can therefore make your own decision (based on your understanding of the system and data) about whether to use the target or estimated asymptote to derive the representativeness value (as you've done).

This was implemented as the default in order to allow for situations where the true asymptote deviates from the UD level. For example, there is now the possibility of introducing the outputs of other home range estimators (e.g., AKDE), the assumption that the inclusion rate (number of out-sample points falling within the UD area) would asymptote at the UD level often doesn't hold and there is no consistent pattern (as far as I'm aware) of the inclusion rate being consistently higher or lower when using other estimators. In short, the thinking went that the important thing is that the information gain is levelling off, not that it asymptotes at any specific level. Now, it is also important to consider that if sample sizes are low to begin with (e.g. <=10) the results are very finicky and need to be taken with a grain of salt. 

All that said, this is a point that needs be better studied from various angles. First, the behavior of the asymptote has mostly been studied when using 50% UDs (the recommended level for important site ID) rather than 95%. Second, the behavior of the asymptote when using different home range estimators (e.g. AKDE). Third, whether there are different functional relationships btwn sample size and inclusion rate for different 'distribution types'.

In summary, this is a good point that requires more investigation. As for your situation, it sounds like it would be better to use the target asymptote of 0.95 for calculating your 'rep' value. If you need code for making a custom plot from the bootTable showing this re-calculated 'rep' value, let me know and I'll send you some.

Hope that clarifies things a bit.

**Comment by MartinBeal** [comment]
Thanks, you make a good point. I'll confer with others and review whether the behavior of defaulting to the estimated asymptote should change. Good luck with your analyses.

**Comment by steffenoppel** [comment]
Thank you Martin for the quick response! Hi Allison - great to hear that
you found our package for your PhD (you did a field season with me on the
North Slope > 10 years ago if I remember correctly...?)!

I agree that there can be cases where you get an asymptote that seems to be
very low. We had such issues with frigatebirds, which travel almost
randomly in any direction and never really congregate in any spot. But it
would be excellent if you could share 1-2 examples from your work, where
you are particularly concerned. If you are not comfortable sharing the raw
data you could share the output table of repAssess (set bootTable==TRUE to
get the table back), just so we can have a look and find a solution.

cheers,
steffen
